azure/template.json

```diff
@@ -707,5 +729,25 @@
         }
     ],
     "outputs": {
+        "ResourceGroupName": {
```

Comment:
This doesn't match the variable name in the release definition (probably best to change release def).  Also looks like the sandbox-template will need to output the same value